static/app/views/releases/list/index.tsx

```diff
@@ -326,7 +350,7 @@ export default function ReleasesList() {
               <DatePageFilter
                 disallowArbitraryRelativeRanges
                 menuFooterMessage={t(
-                  'Changing this date range will recalculate the release metrics.'
+                  'Changing this date range will recalculate the release metrics. Since only specific date ranges are allowed, we recommend selecting from one of the predefined options above.'
```

Comment:
Nit!! The word "Since" is bugging me here. When i read it it seems like there's a reason why only specific ranges are allowed, but we're not explaining why (we don't really have space to explain).
I think if it was a little more factual sounding then it reads better? idk

```suggestion
                  'Changing this date range will recalculate the release metrics. Select a supported date range from the list above.'
```